images/fcos-bastion-image/root/usr/bin/dnsmasq-init.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+PODMAN_DNSMASQ_BASEDIR=/var/opt/dnsmasq
```

Comment:
isn't this set by the environment file too?

```suggestion
PODMAN_DNSMASQ_BASEDIR=${PODMAN_DNSMASQ_BASEDIR:-/var/opt/dnsmasq}
```

Reply:
Its not needed, so removed it.